permissions/plugin-qualys-api-security.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "qualys-api-security"
+github: "jenkinsci/qualys-api-security-plugin"
+paths:
+- "com/qualys/plugins/qualys-api-security-plugin"
```

Comment:
The `paths` item is your pom's `groupId` with periods replaced with `/` plus your `artifactId`, which in this case is `qualys-api-security`. So, you will need to remove `-plugin` from your path.

Reply:
Thanks for pointing that out @slide ! Appreciate your attention to details.

Reply:
removed `-plugin` from paths.